src/db.py

```diff
@@ -0,0 +1,34 @@
+__author__ = 'Tsintsir'
+
+from binascii import b2a_hex
+from os import urandom
+from pymongo import MongoClient
+from pbkdf2 import PBKDF2
+
+
+def create_user(username, email, password):
```

Comment:
Certainly, feel free to defend or challenge any decisions you want to make regarding this; I'm just going off of my experience, but there's no reason that it's more correct or valuable than yours!